apps/web/scripts/installExternalCourses.py

```diff
@@ -0,0 +1,20 @@
+import json
+import sys
+import os
+
+def main():
```

Comment:
I wonder, now that we are getting sophisticated enough to write Python code for this functionality, is there a way we could perhaps use some existing solution for this problem.

Looks like this is basically a package management issue so I'm wondering whether it would be a good idea to use an existing package manager for this? :thinking:

I think that the courses could be published to PyPI probably :thinking:  the only issue I see perhaps is that we would need to keep updating the lockfile in order to deploy the latest version. But if there's a non-hacky way to deal with that the it should be ok :thinking:

Reply:
I don't think we need to assume that the course authors know PyPI or even Python.

Reply:
that's true, but the packaging can be done automatically, similarly how linting can be done automatically. Basically, the CI tool can automatically publish a new version on the `main` branch if no issues were detected